weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/tabularListViews/operators.ts

```diff
@@ -41,6 +41,13 @@ export const operationConverter = (
         substr: {$literal: item.value},
       },
     };
+  } else if (item.operator === '(string): by') {
```

Comment:
i presume this operator is for `monitored "by" XYZ`. Seems weird to have the `(string)` prefix if so